File: 151004/Ivanenko/DC_Lab2/DC_Lab2/DTO/StickerResponseTo.cs
using DC_Lab1.DTO.Interface;

namespace DC_Lab1.DTO
{
    public class StickerResponseTo(int Id, string? Name) : IResponseTo
    {
        public int Id { get; set; } = Id;

        public string? Name { get; set; } = Name;
    }
}

File: 151001/Timoshek/publisher/src/main/java/by/bsuir/egor/dto/StickerResponseTo.java
package by.bsuir.egor.dto;

public class StickerResponseTo {
    private long id;
    private String name;
    private long issueId;

    public void setIssueId(long issueId) {
        this.issueId = issueId;
    }

    public long getIssueId() {
        return issueId;
    }

    public long getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public void setId(long id) {
        this.id = id;
    }

    public void setName(String name) {
        this.name = name;
    }
}
